examples/vision/md/randaugment.md

```diff
@@ -0,0 +1,421 @@
+# RandAugment for Image Classification for Improved Robustness
+
+**Author:** [Sayak Paul](https://twitter.com/RisingSayak)<br>
+**Date created:** 2021/03/13<br>
+**Last modified:** 2021/03/13<br>
+**Description:** RandAugment for training an image classification model with improved robustness.
```

Comment:
Ok, let's emphasize `automated data augmentation` (it's Quoc Le's team, so no wonder they're applied ML to ML 😃 I've just scanned the paper.

```suggestion
**Description:** Automated data augmentation with RandAugment for image classification.
```

Reply:
This might be even more confusing as there's already something called AutoAugment: [AutoAugment: Learning Augmentation Policies from Data](https://arxiv.org/abs/1805.09501)

Reply:
It's up to you 😄 It was just a suggestion.

I scanned the paper and, like many of Quoc Le team's breakthroughs recently, I think this one was about "AutoML" or something close to meta-learning (ML to ML). And yes, the authors compare RandAugment with AutoAugment (where the capitalization is used because it the name of the method).

I'd at least squeeze `data augmentation` into the title, otherwise it may appear to be vague ("improved robustness").

(e.g. "Data augmentation with RandAugment for image classification" - the title is short, the description is longer)

Reply:
I'm super happy you made this example - RandAugment is something I should add to my toolkit! 👍

Reply:
Actually, I even find the "automated" part to be a bit confusing even in RandAugment. Because we are essentially sampling from choices of augmentation transforms from a pre-defined set of transformations with a specific intensity value to apply those transformations. So, there's nothing that automated in this case I think. 

> I'd at least squeeze data augmentation into the title, otherwise it may appear to be vague ("improved robustness").

I think a bit otherwise, though. The name RandAugment itself tells that it has something to do with data augmentation.  On the other hand, there's enough empirical evidence that shows RandAugment improves the robustness of vision models on a general basis. Hence, I don't find "improved robustness" vague.

Reply:
> The name RandAugment itself tells that it has something to do with data augmentation. 

OK. I'd remember to think about new users who aren't familiar with most of the advanced tricks. To them, AutoAugment or even MultiHeadedAttention would be not as familiar, as to those who play with that code on the weekly basis.

So, personally, I'd highly recommend

```
**Description:** Data augmentation with RandAugment for image classification. 
```

which would explain new users that the example is about `data augmentation` using `RandAugment` for task `image classification`.

But that's just me 😄 

Thanks again for putting this example together! It's awesome.